install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -14,6 +14,9 @@ citadelHealthCheck: false
 # 90*24hour = 2160h
 workloadCertTtl: 2160h
 
+# Citadel default behavior if the ca.istio.io/managed namespace label not found
```

Comment:
Can we be consistent with what injector uses ? 
Also I think we agreed that it won't be a boolean in the label.

Reply:
this specifies behavior when no label is found, what citadel is supposed to do.

the actual label is not using boolean.